db/migrate/20200127120545_migrate_data_to_user_dishes.rb

```diff
@@ -0,0 +1,7 @@
+class MigrateDataToUserDishes < ActiveRecord::Migration[5.0]
+  def up
+    Dish.all.each do |dish|
```

Comment:
just FYI:
I would write `Dish.all.pluck(:user_id, :dish_id).each do |user_id, dish_id|`
since it's a little faster.


Why `new(...).save` instead of `create` ?

I think we should also use `save!/create!` in transaction, to minimize the chance of data inconsistency.